### PR TITLE
[1.17] Add logic for running openshift e2e-aws tests

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@mrunalp @haircommander PTAL

**Comment by haircommander** [comment]
weird that the test failed...

/retest

(e2e_rhel run: https://prow.svc.ci.openshift.org/view/gcs/origin-federated-results/pr-logs/pull/cri-o_cri-o/3080/test_pull_request_crio_e2e_rhel/8958/), cross posting in case it fails again

**Comment by rhatdan** [comment]
LGTM

**Comment by saschagrunert** [comment]
> weird that the test failed...
> 
> /retest
> 
> (e2e_rhel run: https://prow.svc.ci.openshift.org/view/gcs/origin-federated-results/pr-logs/pull/cri-o_cri-o/3080/test_pull_request_crio_e2e_rhel/8958/), cross posting in case it fails again

I saw this yesterday as well, might be caused of upstream changes?

File: images/os/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM fedora:29 AS build
```
Comment (TomSweeneyRedHat) [inline images/os/Dockerfile]: should this be bumped up to a new version or perhaps `:latest`?

**Comment by TomSweeneyRedHat** [comment]
One small question, otherwise LGTM.  Given the green test buttons and the fires in the CI as of late, I'm fine with a merge now and minor adjustments (if needed) later.

File: images/os/Dockerfile (hunk 1 above)
Comment (umohnani8) [reply images/os/Dockerfile]: Could be, but I am worried of breaking things that already work :D 
The other repos using similar tests still have it at 29, so just going to leave that for now.

**Comment by umohnani8** [comment]
@rhatdan @saschagrunert can I please get a lgtm label here

**Comment by rhatdan** [comment]
@umohnani8 Is master branch up to at least fedora 30?

**Pull request merged**